app/models/changeset/commit.rb

```diff
@@ -25,7 +25,6 @@ def author
 
   def summary
     summary = @data.commit.message.split("\n").first
-    summary.truncate(80)
   end
```

Comment:
need to check where that summary is used ... could either change everything to `summary.truncate(80)` or extract a `summary_long` to use internally

Reply:
ok, `summary_long` created 🙏🏼